doc/source/whatsnew/v0.17.0.txt

```diff
@@ -87,9 +87,12 @@ Backwards incompatible API changes
 
 .. _whatsnew_0170.api_breaking.other:
 
-.. _whatsnew_0170.api_breaking.convert_objects:
+
 Changes to convert_objects
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _whatsnew_0170.api_breaking.convert_objects:
```

Comment:
ah, possibly, but the other tag  (`_whatsnew_0170.api_breaking`) should actually be put after your new part (it belongs to the next title, you just put the new entry in between of it)